Rocket.Chat/Controllers/Subscriptions/NewRoomViewController.swift

```diff
@@ -167,30 +167,33 @@ class NewRoomViewController: BaseViewController {
             readOnly: readOnlyRoom
         )
 
-        API.current()?.fetch(request, succeeded: { result in
-            guard
-                let success = result.success,
-                let name = result.name, success == true,
-                let auth = AuthManager.isAuthenticated()
-            else {
-                completion(false, result.error)
-                return
-            }
+        API.current()?.fetch(request) { response in
```

Comment:
[weak self]

Reply:
self is not used inside this closure, so it would generate a warning